blobs.go

```diff
@@ -61,6 +61,12 @@ type Descriptor struct {
 	// depend on the simplicity of this type.
 }
 
+// Descriptor returns the descriptor, to make it satisfy the Describable
```

Comment:
This does not satisfy the `Describable` interface. `Describable` must describe itself (in this case, the descriptor), not the thing it is describing.